hash/msx1_cart.xml

```diff
@@ -16163,31 +16165,146 @@ legacy FM implementations cannot find it.
 	</software>
 
 
+	<!-- Kanji cartridges -->
+
+	<software name="hxm200">
+		<description>Toshiba Kanji ROM Cartridge HX-M200 (Japan)</description>
+		<year>1984</year>
+		<publisher>Toshiba</publisher>
+		<info name="alt_title" value="東芝漢字ROMカートリッジ" />
+		<part name="cart" interface="msx_cart">
+			<feature name="slot" value="kanji" />
+			<dataarea name="kanji" size="131072">
+				<rom name="kanjifont.rom" size="131072" crc="d23d4d2d" sha1="db03211b7db46899df41db2b1dfbec972109a967" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="ec700">
+		<description>EC-700 Word Processor (EC-701 + EC-702) (Japan)</description>
+		<year>1985</year>
+		<publisher>Konami</publisher>
+		<info name="alt_title" value="日本語ワードプロセッサ" />
+		<!-- Word processor -->
+		<part name="ec701" interface="msx_cart">
+			<feature name="slot" value="ec701" />
+			<dataarea name="rom" size="524288">
+				<rom name="ec701_program.ic1" size="32768" crc="1caec27b" sha1="4cc5be7536c11a89b5ec736a62a2b832361cd23d" />
+				<!-- should be 3 roms at ic2, ic3, and ic4 -->
+				<rom name="ec701_dictionary.rom" size="393216" crc="b3582eb0" sha1="661c7261dc261e9ef2106755946f95d8b9b699b4" status="baddump" offset="0x20000" />
+			</dataarea>
+		</part>
+		<!-- Kanji ROM -->
+		<part name="ec702" interface="msx_cart">
+			<feature name="slot" value="kanji" />
+			<dataarea name="kanji" size="131072">
+				<rom name="kanjifont.rom" size="131072" crc="3b8fdf44" sha1="fc71561a64f73da0e0043d256f67fd18d7fc3a7f" />
+			</dataarea>
+		</part>
```

Comment:
Same concerns with this one.

Reply:
Yes, it will be automounted when started using "<drivername> ec700".
However since there are no msx1 machine with a kanji rom builtin that is actually convenient since the software will just work (as long as the machine has 2 cartridge slots).
For msx2/2+ machines with a builtin kanji rom, yes, that will override the internal kanji rom. So, yes, for some drivers you would want it to be automounted, for some not.
I could change it into 2 separate software list entries so the end user gets to decide which of the cartridges to mount, but then you lose the documentation that these 2 cartridges were part of a single software package.